### DISCUSSION TITLE
...Is it possible to make custom themes?

### DISCUSSION THREAD

**Discussion by corysimmons** [facebook/docusaurus#9952] [Q&A]
Like is this exposed as some sort of config option like:

```js
// docusaurus.config.js
export default {
  themeConfig: {
    colors: {
      primary: '#ff0000',
      accent: '#0000ff',
      // etc...
    }
  }
}
```

Or do we just get dark and light? :)

**Comment by Josh-Cena** [comment]
I'm not sure what you want. Do you want:

- A new Docusaurus theme plugin (as in, doesn't use any of the default Docusaurus UI components)?
- A new color mode separate from dark/light?
- A new color palette?

They have drastically different levels of difficulties.

**Reply by corysimmons** [reply]
The ability to feed my own custom color scheme into Docusaurus.
